app/views/exchanges/hbx_profiles/_marketplace_plan_year.html.slim

```diff
@@ -0,0 +1,28 @@
+.top-pd.row#inbox_provider_form
+  .col-lg-12
+    nav[aria-label="breadcrumb"]
+      ol.breadcrumb
+        li.breadcrumb-item= link_to l10n('marketplaces.breadcrumb.marketplace'), main_app.issuer_index_exchanges_hbx_profiles_path, remote: true
+        li.breadcrumb-item= link_to l10n('marketplaces.breadcrumb.plan_year'), marketplace_plan_years_exchanges_hbx_profiles_path(year: params[:year]), remote: true
+        li.breadcrumb-item.active[aria-current="page"]
+          = l10n('marketplaces.breadcrumb.carrier')
+  .col-lg-12
+    h1.heading-text= l10n('marketplaces.carriers.title', year: 2024)
```

Comment:
Should this hard-coded `year` be coming from `params[:year]` or some other dynamic source?

Reply:
Oops. you right